Description: fix spelling error (secion -> section) in darkice.cfg man page
Author: Marc Bigler <marc@towards.ch>
Last-Update: 2019-03-23
Index: darkice-1.3/man/darkice.cfg.5
===================================================================
--- darkice-1.3.orig/man/darkice.cfg.5
+++ darkice-1.3/man/darkice.cfg.5
@@ -4,7 +4,7 @@ darkice.cfg \- configuration file for da
 .SH DESCRIPTION
 .PP
 The configuration file consists of sections, with key = value pairs
-separated with spaces and/or tabs inside each secion:
+separated with spaces and/or tabs inside each section:
 
 .nf
 [section1]
